Gruntfile.js

```diff
@@ -5,9 +5,13 @@ module.exports = function(grunt) {
   grunt.initConfig({
     pkg : grunt.file.readJSON('package.json'),
 
-    src_dir_js : [ 'src/main/resources/public/js' ],
+    build_dir_js: 'build/resources/main/public/js',
 
-    build_dir_js : [ 'build/resources/main/public/js' ],
+    gruntfile_js: 'Gruntfile.js',
+
+    src_app_js: 'src/main/resources/public/js/app.js',
+
+    src_files_js: 'src/main/resources/public/js/*.js',
```

Comment:
``src_dir_js`` was ``src/main/resources/public/js`` and then in the option the ``/**`` was added. I desided to create an new variable ``src_files_js`` with include ``*.js``. In my tests it was working fine. Also subfiles were checked, but I will review this again.